test/e2e/framework/expect.go

```diff
@@ -90,3 +90,23 @@ func ExpectHaveKey(actual interface{}, key interface{}, explain ...interface{})
 func ExpectEmpty(actual interface{}, explain ...interface{}) {
 	gomega.ExpectWithOffset(1, actual).To(gomega.BeEmpty(), explain...)
 }
+
+// ExpectGreaterThan expects actual is greater than extra
+func ExpectGreaterThan(actual interface{}, extra interface{}) {
+	gomega.Expect(actual).Should(gomega.BeNumerically(">", extra))
+}
+
+// ExpectGreaterThanOrEqualTo expects actual is greater than or equal to extra
+func ExpectGreaterThanOrEqualTo(actual interface{}, extra interface{}) {
+	gomega.Expect(actual).Should(gomega.BeNumerically(">=", extra))
+}
+
+// ExpectLessThan expects actual is less than extra
+func ExpectLessThan(actual interface{}, extra interface{}) {
+	gomega.Expect(actual).Should(gomega.BeNumerically("<", extra))
+}
+
+// ExpectLessThanOrEqualTo expects actual is less than or equal to extra
+func ExpectLessThanOrEqualTo(actual interface{}, extra interface{}) {
+	gomega.Expect(actual).Should(gomega.BeNumerically("<=", extra))
+}
```

Comment:
Please use gomega directly instead of extending `expect.go`

Reply:
Ack, removed the commit where additional methods were added in order to encourage the use of gomega directly.